### DISCUSSION TITLE
[Technical Questions] Issue with PR after merging

### DISCUSSION THREAD

**Discussion by EmilyMacias** [CS342/.github#51] [Technical Questions] [answered]
### In what area do you have a technical challenge?

Xcode, Simulator & Previews

### Description

Hello! I am having some issues with the merging of my PR, specifically it looks like it's failing the workflow run. Here is an image that shows what's happening:
![Screenshot (712)](https://github.com/CS342/.github/assets/99148529/084baa66-d707-440f-9017-6e1ccb9bc152)

### Reproduction

I received an email about the test failing. You can view it here: https://github.com/CS342Industry/2024-Nourish/actions/runs/7620466606

### Expected behavior

All of the tests passed in my PR (the 7/7) so I am not sure what exactly is failing. I would appreciate any guidance on how to fix the issue and understand what is failing. Thank you!

### Code of Conduct

- [X] I agree to follow this course's Code of Conduct and Contributing Guidelines

**Comment by PSchmiedmayer** [comment, marked as answer]
@EmilyMacias Thank you for sharing this issue! This build failure happed because we were updating our Build Agents right when your build was running. It seems like the build agent went offline right when the upload artifact build was running.

If you experience something like this, I would suggest to try re-running all failed builds. You can find this right in the build UI:
<img width="1624" alt="Screenshot 2024-01-22 at 9 23 59 PM" src="https://github.com/CS342/.github/assets/28656495/d35e199e-d720-45d7-86d4-69ad561e5086">

**Reply by EmilyMacias** [reply]
That worked! Thanks for the advice!
